environments/icds-cas/public.yml

```diff
@@ -166,6 +166,29 @@ additional_syslog_config:
 
 kafka_log_retention: 504 # 21 days
 
+# Promethues,Alertmanager,grafana,exporter settings
+prometheus_db_dir: "{{ encrypted_root }}/prometheus"
+prometheus_storage_retention: "365d"
+prod_mode: True
+alertmanager_external_url: "http://{{ ansible_host }}:9093"
+alertmanager_slack_api_url: "{{ secrets.alertmanager_slack_api_url }}"
+alertmanager_receivers:
+- name: slack
+  slack_configs:
+  - send_resolved: true
+    channel: '#alerts'
```

Comment:
should this be `#alerts-icds`?

Reply:
These are required dummy vars for 3rd party role. i'm overriding these in https://github.com/dimagi/commcare-prometheus/blob/master/roles/alertmanager/templates/alertmanager.yml.j2